### PR TITLE
Add ner case-agnostic config

### PR DESCRIPTION
**Pull request by vaskonov**
Add case-agnostic distil NER for DREAM

### CODE REVIEW

File: deeppavlov/configs/ner/ner_mbert_dream_distil.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            "MODEL_PATH": "{MODELS_PATH}/ner/ner_mbert_dream_distil"
+        },
+        "download": [
+            {
+                "url": "http://files.deeppavlov.ai/v1/ner/ner_mbert_dream_distil.tar.gz",
+                "subdir": "{MODELS_PATH}"
+            }
+        ]
+    }
+}
```
Comment (IgnatovFedor) [inline deeppavlov/configs/ner/ner_mbert_dream_distil.json:155]:
```suggestion
}

```

File: deeppavlov/configs/ner/ner_mbert_dream_distil.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        "dataset_name": "conll2003",
+        "provide_pos": false
+    },
+    "dataset_iterator": {
+        "class_name": "data_learning_iterator"
+    },
+    "chainer": {
+        "in": [
+            "x"
+        ],
```
Comment (IgnatovFedor) [inline deeppavlov/configs/ner/ner_mbert_dream_distil.json:13]:
```suggestion
        "in": ["x"],
```
And edit other lists where newline isn't necessary. I understand that this is automatic json formatting, but it's less readable than incorrect formatting.

File: docs/features/models/ner.rst [hunk 3]
```diff
@@ -41,6 +41,9 @@ Here is the list of all available configs:
     +------------------------------------------------------------------------+--------------------+          +-----------------+------------+------------+
     | :config:`ner_conll2003_bert <ner/ner_conll2003_bert.json>`             | CoNLL-2003         |          | 400 MB          |   1.3 GB   |   91.7     |
     +------------------------------------------------------------------------+--------------------+----------+-----------------+------------+------------+
+    | :config:`ner_mbert_dream_distil <ner/ner_mbert_dream_distil.json>`     | CoNLL-2003         | En+Ru    | 700 MB          |   1.6 GB   |   89.4     |
+    |                                                                        | Collection3        | En+Ru    | 700 MB          |   1.6 GB   |   96.4     |
```
Comment (IgnatovFedor) [inline docs/features/models/ner.rst:45]:
```suggestion
    |                                                                        | Collection3        |          |                 |            |   96.4     |
```

File: docs/features/models/ner.rst [hunk 4]
```diff
@@ -41,6 +41,9 @@ Here is the list of all available configs:
     +------------------------------------------------------------------------+--------------------+          +-----------------+------------+------------+
     | :config:`ner_conll2003_bert <ner/ner_conll2003_bert.json>`             | CoNLL-2003         |          | 400 MB          |   1.3 GB   |   91.7     |
     +------------------------------------------------------------------------+--------------------+----------+-----------------+------------+------------+
+    | :config:`ner_mbert_dream_distil <ner/ner_mbert_dream_distil.json>`     | CoNLL-2003         | En+Ru    | 700 MB          |   1.6 GB   |   89.4     |
```
Comment (IgnatovFedor) [inline docs/features/models/ner.rst:44]: Rename config according naming agreement. I'm not sure about writing both dataset names like `conll2003_collection`, but `dream` should be removed for sure, and maybe `distil` replaced with `distilbert`. Don't forget to rename model weight file and md5 file.

File: docs/features/models/ner.rst [hunk 5]
```diff
@@ -297,6 +301,19 @@ The model also can be trained from scratch by using the command:
...
+Although capitalisation is an important feature for the Named Entity Recognition (NER) task, 
+the NER input data is not always cased, for example, virtual assistants data coming from ASR. 
+Moreover, while developing virtual assistants there is often a need to support interaction in several languages. 
+It has been shown that multilingual BERT can be successfully used for cross-lingual transfer, 
+performing on datasets in various languages with scores comparable to those obtained with language-specific models.  
+
+
+The model :config:`ner_mbert_dream_distil <ner/ner_mbert_dream_distil.json>` was trained on 
+on a concatenation of original and lowered datasets to solve the task. Our model achieves 
+the highest average result on CoNLL-2003 and Collection 3 datasets while being robust to missing casing.
```
Comment (IgnatovFedor) [inline docs/features/models/ner.rst:316]: `the highest` sounds like SOTA.

**Review by IgnatovFedor — CHANGES_REQUESTED** [review]

File: docs/features/models/ner.rst (hunk 4 above)
Comment (vaskonov) [reply docs/features/models/ner.rst:44]: @dilyararimovna how do you think we should name the config?

**Review by IgnatovFedor — APPROVED** [review]

**Pull request merged**